tests/circuitpython-manual/synthio/wave/wave.py

```diff
@@ -0,0 +1,546 @@
+"""Stuff to parse WAVE files.
+
+Usage.
+
```

Comment:
Did you write this file?

Reply:
no, several of the files here are copied verbatim from cpython's stdlib.

Reply:
I added reuse-style license notices to the copied files.